src/main/java/dev/waterdog/ProxyServer.java

```diff
@@ -221,13 +221,14 @@ public boolean handlePlayerCommand(ProxiedPlayer player, String message) {
     }
 
     public boolean dispatchCommand(CommandSender sender, String message) {
-        DispatchCommandEvent event = new DispatchCommandEvent(sender, message);
-        this.eventManager.callEvent(event);
+        String[] args = message.split(" ");
+        DispatchCommandEvent event = new DispatchCommandEvent(sender, args[0], Arrays.copyOfRange(args, 1, args.length));
```

Comment:
To save some performance I would store `Arrays.copyOfRange(args, 1, args.length)` as variable so we can reuse it in `handleCommand()`.

Reply:
Ah yes, you're rigth, 'll do that